contracts/ERC20Safe.sol

```diff
@@ -275,27 +275,19 @@ contract ERC20Safe is BridgeRole, Pausable {
         - lastUpdatedTimestamp
         - depositsCount
     */
-    function getBatch(uint256 batchNonce) public view returns (Batch memory) {
+    function getBatch(uint256 batchNonce) public view returns (Batch memory, bool) {
```

Comment:
@dragos-rebegea could we consider naming the return value to avoid ambiguity? `returns (Batch memory, bool isFinal)`?

Reply:
done